### PR TITLE
Feature/4698 fix purgecss

### LINKED ISSUE #4698: Homepage styles missing for interactive components
### Summary

After purgecss was added, we lost styles for the open nav state and search results—the interactive components that weren't part of the html sample.

Those interactive components should have been visually unchanged from production.

### Frequency

Every time

### How to Reproduce

Look at the homepage from the develop branch—try to hover over the nav, do a quick search

### Device, browser, etc

All

### Screenshots

This is the open state for the "Campaign finance data":
image

Search suggestions:
image

### PR DESCRIPTION
**Pull request by rfultz**
## Summary

- Resolves #4698 

After adding purgecss, we lost styles for the interactive parts of the homepage—the menu wouldn't open and search suggestions were jumbled

### Required reviewers

2

## Impacted areas of the application

The homepage, mostly the interactive components

## Screenshots

The before shots are in the ticket #4698 . The 'after' should look like Production

## Related PRs

Nah

## How to test

- pull the branch
- `npm i` just in case
- `npm run build-sass`
- check [localhost homepage](http://127.0.0.1:8000) to make sure it looks like Prod
- check the interactive parts, too
   - main nav - wide
   - main nav - small
   - search results (make sure you get committee suggestions to see the active/inactive dots)
   - feedback
   - glossary
   - maps
- `npm run build`
   - check the same components
- let me know if things are ugly or invisible

### CODE REVIEW

File: fec/home/templates/purgecss-homepage/toggled.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <div id="feedback" class="js-feedback-box feedback" aria-hidden="true">
+        <button class="js-feedback button--down feedback__close" tabindex="-1"><span class="u-visually-hidden">Close</span></button>
+        <div class="js-status" aria-hidden="true">
+          <div class="message js-message">
+          </div>
+          <ul class="list--buttons">
+            <li><button class="js-reset button--cta-primary feedback__button" type="button" tabindex="-1">Submit another issue</button></li>
+          </ul>
+        </div>
+        <form id="feedback-form" class="container">
```
Comment (patphongs) [inline fec/home/templates/purgecss-homepage/toggled.html:68]: @rfultz Can you incorporate HTML for the recaptcha badge to the feedback, right now it's overlapping.

Screen Shot 2021-06-08 at 3 42 22 PM

**Comment by JonellaCulmer** [comment]
@rfultz I can also see your presidential map replacement test on dev. Should I be able to?

File: fec/home/templates/purgecss-homepage/toggled.html (hunk 1 above)
Comment (patphongs) [reply fec/home/templates/purgecss-homepage/toggled.html:68]: Looks fixed!

[the remark this answers is not in the document]
Comment (patphongs) [reply fec/home/templates/purgecss-homepage/toggled.html:68]: Looks fixed! I'll try out the success state when we push up to dev

**Comment by rfultz** [comment]
@patphongs @JonellaCulmer I also snuck in the fix for the mobile menu locking the body from scrolling. To test:
1. shrink the window < 860px wide
1. scroll the window to make sure you can
1. open the main menu
1. resize the window > 860px
1. this fix should let the window scroll again (it was previously locked until shrinking the window, closing the menu, then expanding again)

**Review by patphongs — APPROVED** [review]
Thanks @rfultz, changes look good!

**Review by JonellaCulmer — APPROVED** [review]
Everything seems to work just fine now. Thanks, @rfultz

**Comment by patphongs** [comment]
@rfultz Tested out the error and success states of the feedback form on stage after merge and it all looks good.

Screen Shot 2021-06-09 at 11 02 16 AM
Screen Shot 2021-06-09 at 11 01 48 AM

**Pull request merged**